### PR TITLE
docs(open-meetings): add 02-29 security office hour minutes

### PR DESCRIPTION
**Pull request by mm73628486283**
## Description

Adding the security office hour meeting minutes for 2024-02-29.

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [ ] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [ ] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

**Review by SebastianBezold — DISMISSED** [review]

File: blog-meeting-minutes/2024-02-29-security-hour.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+slug: security-office-hour-2024-02-29
+title: Security Office Hour 29.02.2024
+authors: 
+    - daniel_dilger
+tags: [meeting-minutes, community, security]
```
Comment (stephanbcbauer) [inline blog-meeting-minutes/2024-02-29-security-hour.md:6]: Is `security` the related tag now we discussed last friday? Can this be used to filter for all security notes?

**Review by stephanbcbauer — COMMENTED** [review]
just a question ...

File: blog-meeting-minutes/2024-02-29-security-hour.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+- Upcoming changes for release 24.05 will focus on FOSS security tools, including
+  - switch from Veracode to CodeQL for SAST,
+  - switch from Gitguardian to gitleaks for secrets scanning,
+  - DAST will not be part of the upcoming TRG until further notice.
+- DAST was removed from TRG due to issues with authenticated scans and SARIF report as scanning alerts in repository security section.
+
+### Open Discussions
+
+- An overarching issue for tracking the tool shifts was discussed, as it is necessary for proper planning by teams.
+  - Teams need to estimate efforts to adjust Github workflowss
```
Comment (tomaszbarwicki) [inline blog-meeting-minutes/2024-02-29-security-hour.md:23]: A typo, I guess:
```suggestion
  - Teams need to estimate efforts to adjust Github workflows
```

**Review by tomaszbarwicki — APPROVED** [review]
LGTM, just a small typo as mentioned above.

**Review by stephanbcbauer — APPROVED** [review]
LGTM

**Review by carslen — APPROVED** [review]

**Pull request merged**